iocore/net/SSLConfig.cc

```diff
@@ -535,3 +522,68 @@ SSLCertificateConfig::release(SSLCertLookup *lookup)
 {
   configProcessor.release(configid, lookup);
 }
+
+void
+SSLTicketParams::LoadTicket()
+{
+  cleanup();
+#if HAVE_OPENSSL_SESSION_TICKETS
+  SSLConfig::scoped_config params;
+  if (REC_ReadConfigStringAlloc(ticket_key_filename, "proxy.config.ssl.server.ticket_key.filename") == REC_ERR_OKAY &&
```

Comment:
Not important, but I personally prefer an empty line between local variable declarations and the code :-).